content/en/docs/started/kubeflow-examples.md

```diff
@@ -5,9 +5,12 @@ weight = 30
                     
 +++
 
-This section introduces the examples in the 
+{{% alert title="Warning" color="warning" %}}
+This section introduces some examples in the 
```

Comment:
This line "This section introduces some examples in the" should probably be outside the warning, as its part of the page.

Reply:
This makes sense. Even further, I simplified the description so we are not confusing users by directing them to the kubeflow/examples repo at the moment. Once we have the up-to-date repository, we can re-introduce kubeflow/examples and remove the warning banner.